src/store.rs

```diff
@@ -14,20 +14,75 @@
  * limitations under the License.
  */
 
+use rocket::futures::TryStreamExt;
+use rocket_db_pools::mongodb::options::{FindOptions, InsertOneOptions};
+use rocket_db_pools::mongodb::Client;
+use rocket_db_pools::mongodb::error::Error;
+use rocket_db_pools::mongodb::bson::doc;
+use rocket_db_pools::mongodb::results::InsertOneResult;
+use rocket_db_pools::mongodb::Collection;
+use rocket::serde::{Deserialize, Serialize};
+
+
 use murmur::MurmurStore;
-use std::fs::File;
+const DB_NAME: &str = "MurmurDB";
+const DB_URI: &str = "mongodb+srv://murmurapi:GuVsTAEbQtNnnbPj@useast.m8j6h.mongodb.net/?retryWrites=true&w=majority&appName=USEast";
+const COLLECTION_NAME: &str = "mmrs";
```

Comment:
I've created another issue to move these to env var https://github.com/ideal-lab5/murmur-api/issues/15